src/transformers/models/encodec/modeling_encodec.py

```diff
@@ -547,6 +586,8 @@ def __init__(self, config: EncodecConfig):
 
         self.quantizer = EncodecResidualVectorQuantizer(config)
 
+        self.commitment_weight = config.__dict__.get("commitment_weight", 1)
```

Comment:
Let's add `commitment_weight` to the configuration, and get this value with `config.commitment_weight`!